ehrql/tables/beta/core.py

```diff
@@ -56,8 +56,10 @@ class patients(PatientFrame):
 
     There is generally a lag between the death being recorded in ONS data and it
     appearing in the primary care record, but the coverage or recorded death is almost
-    complete and the date of death date is usually reliable when it appears. You can
-    find out more about the accuracy of date of death recording in primary care in:
+    complete and the date of death date is usually reliable when it appears. There is
+    also a lag in ONS death recording (see [below](https://docs.opensafely.org/ehrql/reference/schemas/beta.core/#ons_deaths)
```

Comment:
Like **Click here** links, **See below** links result in poor UX for people using screen readers. More generally, they also make it hard for people to know where they will go after they click the link.

```suggestion
    also a lag in ONS death recording (see [`ons_deaths`](/reference/schemas/beta.core/#ons_deaths) below
```